files/openshift_backup.sh

```diff
@@ -34,3 +34,10 @@ for project in $(oc get projects -o name | sed 's#projects/##'); do
   /opt/adfinis/bin/project_export.sh "${project}" &>> "${BACKUP_LOG}"
   /opt/adfinis/bin/non-namespaced_export.sh &>> "${BACKUP_LOG}"
 done
+
+# clean up old backups
+
+if [ ${CLEANUP} -gt 0 ] 
+  then
+  find ${VAULT} -mtime +${CLEANUP} -not -name default.conf -delete
```

Comment:
In another project we're using the following command to directly delete the backup directories, which might be safer with regard to other files in the backup vault:

```bash
find "${VAULT}" -mindepth 1 -maxdepth 1 -name '20*' -daystart -type d -mtime +${RETENTION} | xargs -r rm -rf
```

Renaming the variable to `RETENTION` would also be a good idea.